tests/ui/cmse-nonsecure/cmse-nonsecure-call/generics.rs

```diff
@@ -1,6 +1,7 @@
 //@ add-minicore
 //@ compile-flags: --target thumbv8m.main-none-eabi --crate-type lib
 //@ needs-llvm-components: arm
+//@ ignore-backends: gcc
```

Comment:
All these new test failures are due to the fact that `--target` is used and we now put `libgccjit.so` in a directory named after the target, so `rustc_codegen_gcc` doesn't find `libgccjit.so`.
Is this the correct way to ignore these tests?
Or are tests annotated with `needs-llvm-compoments` not expected to pass with a codegen that is not LLVM?